nwb_conversion_tools/datainterfaces/ecephys/cellexplorer/cellexplorerdatainterface.py

```diff
@@ -17,28 +24,51 @@ class CellExplorerSortingInterface(BaseSortingExtractorInterface):
     def __init__(self, file_path: FilePathType):
         super().__init__(spikes_matfile_path=file_path)
         self.source_data = dict(file_path=file_path)
+        spikes_matfile_path = Path(file_path)
 
         session_path = Path(file_path).parent
         session_id = session_path.stem
-        spikes_cellinfo_file_path = session_path / f"{session_id}.spikes.cellinfo.mat"
-        if spikes_cellinfo_file_path.is_file():
-            cell_info = loadmat(spikes_cellinfo_file_path).get("spikes", np.empty(0))
-            cell_info_fields = cell_info.dtype.names
+
+        assert spikes_matfile_path.is_file(), f"The spikes_matfile_path ({spikes_matfile_path}) must exist!"
+
+        try:
+            spikes_mat = scipy.io.loadmat(file_name=str(spikes_matfile_path))
+            self.read_spikes_info_with_scipy = True
+        except NotImplementedError:
+            spikes_mat = hdf5storage.loadmat(file_name=str(spikes_matfile_path))
+            self.read_spikes_info_with_scipy = False
+
+        cell_info = spikes_mat.get("spikes", np.empty(0))
+        self.cell_info_fields = cell_info.dtype.names
+
+        if self.read_spikes_info_with_scipy:  # Logic for scipy
```

Comment:
```suggestion
        if self.read_spikes_info_with_scipy:
```

Sufficiently clear without extra comment